README.rst

```diff
@@ -13,28 +13,28 @@
 |Coverage Status|
 
 Zipline is a Pythonic algorithmic trading library. It is an event-driven
-system that supports both backtesting and live-trading. Zipline is currently used in production as the backtesting and live-trading
+system that for backtesting. Zipline is currently used in production as the backtesting and live-trading
```

Comment:
"It is an event-drive system that for" -> "It is an event driven system for backtesting"